packages/dev/core/src/Materials/material.ts

```diff
@@ -1828,13 +1828,15 @@ export class Material implements IAnimatable, IClipPlanesHolder {
      */
     // eslint-disable-next-line @typescript-eslint/naming-convention
     private releaseVertexArrayObject(mesh: AbstractMesh, forceDisposeEffect?: boolean) {
-        if ((<Mesh>mesh).geometry) {
-            const geometry = <Geometry>(<Mesh>mesh).geometry;
+        const geometry = <Geometry>(<Mesh>mesh).geometry;

```

Comment:
You can remove the `Geometry` cast, as you moved the code before the `if` check.

Reply:
Done

Reply:
You should remove the `Geometry` import to fix the CI error.

Reply:
Yep, sorry about that.